vendor/assets/javascripts/data-confirm-modal.js

```diff
@@ -37,6 +37,8 @@
 
   var defaults = {
     title: 'Are you sure?',
+    titleClass: 'modal-title',
```

Comment:
Any reason why it should be possible to customize `modal-title` (which is required by Bootstrap 3, 4, 5), but not for example `modal-body`, `modal-header`, `modal-footer`?

My best guess is that a developer should be able to specify `<h4 class="h5"></h4>` to preserve the style. Is that correct?

Reply:
Yes Tagliala, that's exactly my idea.
Although, in the end devs could override the .modal-title class or reach the element in another way through css this would help big time not re writting CSS.